src/containers/Welcome/index.js

```diff
@@ -20,31 +26,58 @@ export default class Welcome extends React.Component {
     }).isRequired,
   };
 
+  state = {
+    inputMarginTop: 104,
+    isFocused: false,
+  };
+
+  scrollUp = () => {
```

Comment:
Let's rename this to `handleFocus` to better illustrate why the code is doing what it's doing. (Same for `scrollDown`).